### PR TITLE
feature: send APP_INITIALIZED telemetry for electron application

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes

This PR is to enable the `APP_INITIALIZED` telemetry event from electron devices.

#### Pull request checklist

- [x] Addresses an existing issue: Fixes AD #1604608
- [x] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/electron/device-connect-view/device-connect-view-initializer.ts [hunk 1]
```diff
@@ -74,4 +76,7 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then((persistedDat
 
     const renderer = new DeviceConnectViewRenderer(ReactDOM.render, document, props);
     renderer.render();
+
+    // send app initialized telemetry - depending on user consent
```
Comment (smoralesd) [inline src/electron/device-connect-view/device-connect-view-initializer.ts]: this comment seems redundant

File: src/electron/device-connect-view/send-app-initialized-telemetry.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+// Licensed under the MIT License.
+import { TelemetryEventHandler } from '../../background/telemetry/telemetry-event-handler';
+import { TelemetryEventSource, TriggeredByNotApplicable } from '../../common/extension-telemetry-events';
+import { APP_INITIALIZED } from '../common/electron-telemetry-events';
+
+export const sendAppInitializedTelemetryEvent = (telemetryEventHandler: TelemetryEventHandler) => {
+    telemetryEventHandler.publishTelemetry(APP_INITIALIZED, {
+        telemetry: {
+            triggeredBy: TriggeredByNotApplicable,
+            source: TelemetryEventSource.ElectronAppInitialized,
```
Comment (smoralesd) [inline src/electron/device-connect-view/send-app-initialized-telemetry.ts]: This should be `TelemetryEventSource.ElectronDeviceConnect` as the Device Connect is the general name for this view (the source doesn't point to the file itself... but to the view in general or components inside the UI)

Comment (Shobhit1) [reply src/electron/device-connect-view/send-app-initialized-telemetry.ts]: Updated. I got an impression that the variable source is suppose to kind of depict the purpose of the telemetry event; but I understand now that it is more about source of the event that is publishing the telemetry.

File: src/electron/device-connect-view/device-connect-view-initializer.ts (hunk 1 above)
Comment (Shobhit1) [reply src/electron/device-connect-view/device-connect-view-initializer.ts]: removed.

**Pull request merged**